keyboards/cmm.studio/saka68/keymaps/default/keymap.c

```diff
@@ -0,0 +1,35 @@
+/* Copyright 2020 CMM.Studio Freather
```

Comment:
This file should be deleted.

Reply:
deleted the whole folder of cmm.studio. now only have cmm_studio folder